src/App.jsx

```diff
@@ -0,0 +1,64 @@
+import { Routes, Route } from 'react-router-dom'
+
+import './App.css'
+
+import Home from "./components/Home/Home"
+import Profile from "./components/Profile/Profile"
+import SinglePostPage from './components/Post/SinglePostPage'
+import Users from './components/Profile/Users'
+import Aside from './components/Aside'
+import Header from './components/Header'
+
+export const baseURL = "https://boolean-api-server.fly.dev/AllyDouillette"
```

Comment:
Good choice. The only change I'd make here is move this to its own file so you don't have to import it from the App file everywhere. I usually create a file called `constants.js`